Sources/ArcGISToolkit/Components/Offline/OfflineMapInfo.swift

```diff
@@ -66,6 +66,11 @@ public extension OfflineMapInfo {
     var portalItemID: Item.ID { .init(info.portalItemID)! }
 }
 
+extension OfflineMapInfo: Identifiable {
+    public var id: Item.ID { portalItemID }
```

Comment:
Could this just replace the `portalItemID` property? It seems like both aren't needed.

Reply:
I don't have a strong opinion on this. I could see it both ways.

Reply:
Updated - see if you guys like this better.